### ISSUE TITLE
Debug why there are large storage accounts on testnet and devnet (inflation related?)

### ISSUE THREAD

**Issue by mvines** [solana-labs/solana#10870]
http://explorer.solana.com/supply?cluster=devnet (similar for testnet) displays:
![image](https://user-images.githubusercontent.com/1281082/86261254-2b481980-bb73-11ea-9129-368778bf0a95.png)
which doesn't really make sense.

Are these accounts just legacy from before the storage program was removed?   Lets:
1. Fully understand where they came from (testnet doesn't maintain a ledger history on GCE, but we do have devnet history so it's possible to search older snapshots if needed)
2. Ensure this can't happen on mainnet-beta !
3. Consider removing these accounts by manually creating a new snapshot without them and hard forking devnet and testnet

**Comment by rwalker-com** [comment]
reward pools were initially created with infinite tokens, they _**were**_ populated during genesis by stake/storage program code, and were excepted from any "total tokens" calculations

**Comment by rwalker-apple** [comment]
ah, nevermind.  I see: is done by the runtime.

**Comment by sakridge** [comment]
These are back at slot 15,754,919

```
5Wwq3cCpHzEMoQifq7yDXJC1RfdbKQS8sgWSCvq3rJP:
  - balance: 18446744073.709553 SOL
  - owner: 'Storage111111111111111111111111111111111111'
  - executable: false
  - data: '5Sxr3'
  - data_len: 4
13pnsN5WU8B4qBfSXUJ1P3apmYnk7uk7J4hVbNgCpPh4:
  - balance: 18446744073.709553 SOL
  - owner: 'Storage111111111111111111111111111111111111'
  - executable: false
  - data: '5Sxr3'
  - data_len: 4
```

So, doesn't seem to be a new occurrence.

**Comment by sakridge** [comment]
Seems that this would have created these 32 storage reward pool accounts:

https://github.com/solana-labs/solana/blob/3415db9739f429f7efda7b0aec6c1d6458ae6b88/programs/storage/src/rewards_pools.rs

**Comment by rwalker-com** [comment]
yeah, but adding each of these to capitalization would result in capitalization moving in a negative direction (-32 to be exact ;) ).  I expect that initial bank and genesis had capitalization right.

**Comment by ryoqun** [comment]
Oh, maybe I'm going to fix this at #11996 :)

**Comment by ryoqun** [comment]
Fix confirmed:

![image](https://user-images.githubusercontent.com/117807/92467692-fe685080-f20c-11ea-9466-d525cbccfaad.png)


```
$ ./target/release/solana epoch-info --url http://testnet.solana.com 

Block height: 28309720
Slot: 34672025
Epoch: 93
Epoch Slot Range: [34652256..35084256)
Epoch Completed Percent: 4.576%
Epoch Completed Slots: 19769/432000 (412231 remaining)
Epoch Completed Time: 2h 11m 47s/2days (1day 21h 48m 12s remaining)
```
